.github/workflows/code_style.yaml

```diff
@@ -8,6 +8,18 @@ on:
       - prerelease_test
   pull_request:
 jobs:
+  perl_checks:
+    name: Check Perl code in tree
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install prerequisites
+        run: sudo apt install perltidy
+      - name: Checkout source
+        uses: actions/checkout@v2
+      - name: Format Perl files, if needed
+        run: find . -name '*.p[lm]' -exec perltidy --profile=scripts/perltidyrc {} +
```

Comment:
`{}` should ideally be escaped like `'{}'`.

Reply:
It is not necessary for any shell and it is standard pattern in, e.g., Emacs for `find`.

Reply:
It appears that it's really hard to find a shell that breaks indeed https://unix.stackexchange.com/questions/8647/gnu-find-and-masking-the-for-some-shells-which
It was suggested in `find`'s documentation though.

Reply:
Ah, good to know. Wonder why they recommend it since it doesn't break any shell I know of. As the post mentions, this was a problem ages ago, but now things are better.